frontend/packages/helm-plugin/src/components/list-page/AdminHelmReleaseListPage.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { Trans, useTranslation } from 'react-i18next';
+import { Link, useParams } from 'react-router-dom-v5-compat';
+import NamespacedPage, {
+  NamespacedPageVariants,
+} from '@console/dev-console/src/components/NamespacedPage';
+import CreateProjectListPage, {
+  CreateAProjectButton,
+} from '@console/dev-console/src/components/projects/CreateProjectListPage';
+import { PageHeading } from '@console/internal/components/utils';
+import HelmReleaseList from './HelmReleaseList';
+
+type PageContentsProps = {
+  namespace: string;
+};
+
+const PageContents: React.FC<PageContentsProps> = ({ namespace }) => {
+  const { t } = useTranslation();
+  return namespace ? (
```

Comment:
We should support showing releases in all namespaces (for users with authority) rather than falling back to the project list in the admin/converged perspective.

Reply:
Agree, I will update it. As an Admin perspective do support All namespaces.